### PR TITLE
More misspellings

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**
See https://github.com/jjhelmus/nmrglue/pull/165.

### CODE REVIEW

**Review by luzpaz — APPROVED** [review]

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -7670,6 +7672,7 @@ controles->controls, controllers,
 controling->controlling
 controll->control
 controllerd->controlled
+controlller->controller
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:7675]: Please add: 
```
controllled->controlled
controlllers->controllers
controllling->controlling
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -8957,6 +8960,7 @@ decomposion->decomposition
 decomposit->decompose
 decomposited->decomposed
 decompositing->decomposing
+decompositon->decomposition
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:8963]: Please add `decompositons->decompositions`

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -13737,6 +13746,7 @@ exploting->exploiting, exploring,
 expoed->exposed
 expoential->exponential
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:13747]: Add `expoent->exponent`

**Review by luzpaz — CHANGES_REQUESTED** [review]

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:7676]: Done.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:8966]: Done.

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:13752]: Done.

**Comment by luzpaz** [comment]
Thanks!

**Pull request merged**